cmd/kepler/main.go

```diff
@@ -105,3 +112,33 @@ func waitForInterrupt(ctx context.Context, signals ...os.Signal) (func() error,
 			cancel()
 		}
 }
+
+func parseArgsAndConfig() (*config.Config, error) {
+	app := kingpin.New("power-monitor", "Power consumption monitoring exporter for Prometheus.")
```

Comment:
"kepler" not  "power-monitor"

Reply:
also the description should be same as for Kepler "Kubernetes Efficient Power Level Exporter"

Reply:
@sthaha you want to change the help string  to "Kubernetes Efficient Power Level Exporter" ?